nuitka/plugins/standard/PlaywrightPlugin.py

```diff
@@ -0,0 +1,168 @@
+            action="append",
+            dest="include_browsers",
+            default=[],
+            help="""\
+            Playwright browser to include. Can be specified multiple times. use "all" to include all installed browsers.
+            """,
+        )
+
+    def _getPlaywrightInfo(self):
+        "Determine the path of the playwright module."
+        try:
+            info = self.queryRuntimeInformationMultiple(
+                info_name="playwright_info",
+                setup_codes="""
+from playwright import __file__ as playwright_file
+from os.path import dirname
+""",
+                values=(("playwright_file", "dirname(playwright_file)"),),
+            )
+        except NuitkaCalledProcessError as e:
+            self.debug("Exception during detection: %r" % e)
+            raise
+
+        if info is None:
+            self.sysexit("Error, it seems 'playwright' is not installed or broken.")
+
+        return info
+
+    def getRegistryDirectory(self):
+
+        env_defined = os.environ.get("PLAYWRIGHT_BROWSERS_PATH")
+        path_home = os.path.expanduser("~")
+        playwright_module_path = self._getPlaywrightInfo().playwright_file
+
+        result = os.path.join(
+            playwright_module_path, "driver", "package", ".local-browsers"
+        )
+        if os.path.exists(result) and next(os.scandir(result), False):
+            return result
+        elif env_defined == "0":
+            return result
+        elif env_defined:
+            result = os.path(env_defined)
+        else:
+            cache_directory = ""
+            if os.name == "posix":
+                cache_directory = os.environ.get(
+                    "XDG_CACHE_HOME", os.path.join(path_home, ".cache")
+                )
+            elif os.name == "darwin":
+                cache_directory = os.path.join(path_home, "Library", "Caches")
+            elif os.name == "nt":
+                cache_directory = os.environ.get(
+                    "LOCALAPPDATA", os.path.join(path_home, "AppData", "Local")
+                )
+
+            result = os.path.join(cache_directory, "ms-playwright")
+
+        if not os.path.isabs(result):
+            init_cwd = os.environ.get("INIT_CWD") or os.getcwd()
+            result = os.path.join(os.path.abspath(init_cwd), result)
+
+        return result
+
+    def get_Installed_Playwright_Browsers(self):
+
+        registry_directory = self.getRegistryDirectory()
+
+        if not os.path.exists(registry_directory):
+            return {}
+
+        browsers_installed = list(os.scandir(registry_directory))
+        for browser in browsers_installed:
+            if browser.name == ".links":
+                browsers_installed.remove(browser)
+                break
+
+        return {browser.name: browser for browser in browsers_installed}
+
+    def considerDataFiles(self, module):
+
+        if module.getFullName() != "playwright" or not self.include_browsers:
```

Comment:
Plugins really should register for module names, and they should be able to like disable themselves, if during init they find there won't be anything to do for me, like if no browsers are to be included. That is not on your plate though.